localstack/services/apigateway/router_asf.py

```diff
@@ -31,7 +31,13 @@ def to_invocation_context(
         url_params = {}
 
     method = request.method
-    path = request.full_path if request.query_string else request.path
+    url_params_stage = url_params.get("stage", "dev")
+    url_params_path = url_params.get("path", request.path)
```

Comment:
Can we assume defaults for non-present stage and path variables or should we fail fast?